### ISSUE TITLE
Limit some builds to two lanes

### ISSUE THREAD

**Issue by clippermadness** [Shippable/support#4814]
We have a number of projects that we build with Shippable. One of them is a build that uses Selenium and Browserstack.

Due to the cost of Browserstack, we'd like to limit the number of concurrent builds for that project to 2, and allow all other projects to build with as many concurrent builds as we purchase from Shippable.

How can we accomplish this?

I have tried using node pools, but I can't find a way to specify in shippable.yml that a particular project may use a node from a set of node pools.

There is a configuration to make all builds for a project serial, but not limit it to some number.

**Comment by trriplejay** [comment]
unfortunately there isn't a way to do this today.  The closest you'd get is allocating two nodes to a nodepool, and dedicate that nodepool to the browserstack tests, however those nodes would become unavailable for use by the other node pools, so it would reduce the concurrent builds for other projects.

**Comment by clippermadness** [comment]
Is there a way to indicate that a particular project may be built on more than just one node pool?

**Comment by trriplejay** [comment]
No, at the moment a project can only be assigned to a single node pool. The only exception is for [matrix jobs](http://docs.shippable.com/ci/matrix-builds/#including-jobs) but that doesn't seem to be quite what you're looking for.
